jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -132,9 +132,9 @@ public void init(final V presenter) {
     }
 
     protected void controlBulkOperations(final ListTable<T> extendedPagedTable) {
-        enableWidgets(Iterables.getFirst(extendedPagedTable.getRightActionsToolbar(),
-                                         null),
-                      extendedPagedTable.hasSelectedItems());
+        enableWidgets(
+                Streams.stream(extendedPagedTable.getRightActionsToolbar()).findFirst().orElse(null),
```

Comment:
@nmirasch we can perhaps avoid using Guava altogether, something like:
```suggestion
                StreamSupport.stream(extendedPagedTable.getRightActionsToolbar().spliterator(),
                                           false).findFirst().orElse(null)
```